src/cfnlint/data/ExtendedSpecs/append_spec/allowed_values.json

```diff
@@ -0,0 +1,240 @@
+{
+  "ResourceTypes": {
+    "AWS::AmazonMQ::Broker": {
+      "Properties": {
+        "EngineVersion": {
+          "AllowedValues": [
+            "5.15.0"
+          ]
+        },
+        "DeploymentMode": {
+          "AllowedValues": [
+            "SINGLE_INSTANCE",
+            "ACTIVE_STANDBY_MULTI_AZ"
+          ]
+        },
+        "EngineType": {
+          "AllowedValues": [
+            "ACTIVEMQ"
+          ]
+        }
+      }
+    },
+    "AWS::AmazonMQ::Configuration": {
+      "Properties": {
+        "EngineVersion": {
+          "AllowedValues": [
+            "5.15.0"
+          ]
+        },
+        "EngineType": {
+          "AllowedValues": [
+            "ACTIVEMQ"
+          ]
+        }
+      }
+    },
+    "AWS::DMS::Endpoint": {
+      "Properties": {
+        "SslMode": {
+          "AllowedValues": [
+            "none",
+            "require",
+            "verify-ca",
+            "verify-full"
+          ]
+        }
+      }
+    },
+    "AWS::EC2::Volume": {
+      "Properties": {
+        "VolumeType": {
+          "AllowedValues": [
+            "gp2",
+            "io1",
+            "sc1",
+            "st1",
+            "standard"
+          ]
+        }
+      }
+    },
+    "AWS::EC2::VPC": {
+      "Properties": {
+        "InstanceTenancy": {
+          "AllowedValues": [
+            "dedicated",
+            "default"
+          ]
+        }
+      }
+    },
+    "AWS::Lambda::Function": {
+      "Properties": {
+        "Runtime": {
+          "AllowedValues": [
+            "nodejs",
+            "nodejs4.3",
+            "nodejs6.10",
+            "nodejs8.10",
+            "java8",
+            "python2.7",
+            "python3.6",
+            "python3.7",
+            "dotnetcore1.0",
+            "dotnetcore2.0",
+            "dotnetcore2.1",
+            "nodejs4.3-edge",
+            "go1.x"
```

Comment:
Need to update for new Lambda (ruby and custom)

Reply:
yeah